pkg/manager/manager.go

```diff
@@ -49,18 +50,20 @@ type KubeMacPoolManager struct {
 	stopSignalChannel        chan os.Signal
 	podNamespace             string
 	podName                  string
+	waitingTime              int
```

Comment:
Please add a docstring explaining why we have this attribute

Reply:
done